### DISCUSSION TITLE
BUG Alkemio Logo

### DISCUSSION THREAD

**Discussion by Roseghost** [alkem-io/alkemio#877] [General] [answered]
When you click the Alkemio logo at the login in page you go to a different page 
![image](https://user-images.githubusercontent.com/64191883/201708812-a044bfd3-008d-4cd9-9f0c-d79597077244.png)

it links to https://alkem.io/about

**Comment by oldhoyte** [comment, marked as answer]
Good catch! I'll add it to the backlog

**Reply by techsmyth** [reply]
Also this should be addressed as part of the social logins work that is being finalized...
